LayoutTests/TestExpectations

```diff
@@ -1348,8 +1348,9 @@ webkit.org/b/126166 [ Debug ] js/dfg-int32array-overflow-values.html [ Skip ]
 webkit.org/b/126166 [ Debug ] js/dfg-uint32array.html [ Skip ]
 webkit.org/b/126166 [ Debug ] js/dfg-uint32array-overflow-values.html [ Skip ]
 
-# This test verifies that a mismatch reftest will fail as intended if both results are same. (introduced in r93187)
+# These tests verify that a mismatch reftest will fail as intended if both results are same.
 fast/harness/sample-fail-mismatch-reftest.html [ WontFix ImageOnlyFailure ]
+fast/forms/input-text-focus-ring-mismatch.html [ WontFix ImageOnlyFailure ]
```

Comment:
The correct way to add a mismatch test is to give the mismatch file an `-expected-mismatch.html` suffix.

So `input-text-focus-ring-mismatch-expected.html` should be renamed to `input-text-focus-ring-expected-mismatch.html`, and the `TestExpectations` file should be unmodified.

Reply:
Fixed. Thanks for guiding me to the right way to add a mismatch ref test.